src/DssDirectDepositHub.sol

```diff
@@ -113,6 +144,14 @@ contract DssDirectDepositHub {
     event Quit(bytes32 indexed ilk, address indexed usr);
     event Exit(bytes32 indexed ilk, address indexed usr, uint256 amt);
 
+    /**
+        @dev sets msg.sender as authed
+        Allows joining/exiting Dai by:
+           - giving infinite approval for Dai on the DaiJoin contract
+           - hoping the DaiJoin in the Vat
+        @param vat_     address of the DSS vat contract
```

Comment:
Regardind `address` or `Address` (whether to capitalize the first letter of the description) - this is uninified throughut the documentation, suggest to align everywhere.